StRoot/StJetMaker/StJetMaker2009.cxx

```diff
@@ -91,8 +91,18 @@ int StJetMaker2009::Make()
       // Keep track of number of good vertices
       int nvertices = 0;
 
+      // Start: bassam
```

Comment:
I would not add such comments in the code as I don't see much value in them. For an onlooker as myself these just add noise to the code when reviewing it